src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -680,6 +680,10 @@ public ErrorDiagnostic ArgumentCountMismatch(int argumentCount, int mininumArgum
                 "An empty indexer is not allowed. Specify a valid expression."
             );
 
+            public ErrorDiagnostic ExpectedBodyStart() => new ErrorDiagnostic(

```

Comment:
[nit] Update the method name to reflect that it's looking for body start or 'if'

Reply:
Changed it to `ExpectedBodyStartOrIf`.